src/test/resources/config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "projectID": "your project ID"
```

Comment:
Is it okay to use "your project ID" here?

Reply:
yes, it's just default file, so you won't need to create file and folder for it manually, you need only change "your project ID" with project id

Reply:
That's OK. An alternative would be giving an example as well. For instance: `"your project ID, e.g. my-gcp-project-123"`